lapis2-docs/src/content/docs/references/introduction.mdx

```diff
@@ -0,0 +1,23 @@
+---
+title: Introduction
+description: Introduction to References
+---
+
+To use LAPIS there are a few things you need to know. This section will cover the basics of how to use LAPIS.
+
+## Endpoints
+
+LAPIS can be accessed by calling different endpoints. Each endpoint has a different purpose and will return different data.
+For example the `GET /sample/aggregated` endpoint will return how many sequences fulfill the provided filter criteria.
+You can find the different endpoints in the [Open API / Swagger](/references/open-api-definition/), alongside a convinient way to test them out using Swagger-UI.
+
+## Filters
+
+Each endpoint can be filtered by different parameters. These parameters are passed as query parameters in the URL.
```

Comment:
```suggestion
Each endpoint can be filtered by different parameters. These parameters are passed as query parameters in the URL or in the body in case of POST....
```